src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -284,6 +285,8 @@ public IEnumerable<ConnectorModel> Connectors
         /// </summary>
         public ObservableCollection<NoteModel> Notes { get { return notes; } }
 
+        public ObservableCollection<AnnotationModel> Annotations { get { return annotations; } }
+
```

Comment:
As we've discussed before, this isn't really the best pattern, but it should be fine for now and keeps things consistent.